data/persistence/src/commonMain/kotlin/social/androiddev/common/persistence/localstorage/DodoAuthStorage.kt

```diff
@@ -24,7 +24,7 @@ interface DodoAuthStorage {
     /**
      * Save the @param token keyed by @param server
      */
-    suspend fun saveAccessToken(server: String, token: String)
+    fun saveAccessToken(server: String, token: String)
```

Comment:
I also changed this method to be sync, as there are no suspend points

Reply:
I think this should be reverted back. Please see my other comment in review on more details